CMakeLists.txt

```diff
@@ -58,6 +58,10 @@ message(STATUS "Found LLVM ${LLVM_PACKAGE_VERSION} in ${LLVM_DIR}")
 list(APPEND CMAKE_MODULE_PATH ${LLVM_DIR})
 
 include(AddLLVM)
+
+set(LLVM_TOOLS_BINARY ${LLVM_INSTALLATION_DIR}/bin)
```

Comment:
Just in case LLVM_DIR has LLVM which has LLVM=$LLVM_PATH/lib/cmake/llvm
In  README we should build f18 as:
LLVM=<LLVM_INSTALLATION_DIR>/lib/cmake/llvm cmake -DLLVM_DIR=$LLVM.
where is this LLVM_INSTALLATION_DIR defined?

Reply:
We are using now LLVM variables